cmd/context/update-kubeconfig.go

```diff
@@ -57,7 +57,7 @@ func UpdateKubeconfigCMD(okClientProvider oktetoClientProvider) *cobra.Command {
 		Hidden: true,
 		Use:    "update-kubeconfig",
 		Args:   utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#kubeconfig"),
-		Short:  "Download credentials for the Kubernetes cluster selected via 'okteto context'",
+		Short:  "Download credentials for the Kubernetes context selected via 'okteto context'",
```

Comment:
from a product perspective @ifbyol do you think it's a good idea to also say "Kubernetes context"?

Reply:
I'm not sure, with `okteto context` you don't select a `Kubernetes context`, you select an `Okteto context`, which is connected to a Kubernetes cluster. So, I wouldn't change it

Reply:
It's also odd to me changing this. @minhoryang can you please revert the changes in reference to kubernetes explicitly?